lib/Conversation/styles.scss

```diff
@@ -116,9 +127,24 @@ $conversation-meta-text-color: $color-brand-dark !default;
 }
 
 .form__actions {
+  position: relative;
   text-align: right;
 
   .form__actions {
     margin: 0 0 0 $layout-spacing-base/2;
   }
+}
+
+.form__drop-down {
+  position: absolute;
+}
+
+.form__button {
```

Comment:
Good job on the styling! I like the modularity of it all 👍 

My only concern is about the namespace of this `.form` component. What do you think about making it more specific in namespace? I've got a feeling that `.form__input` and `.form__actions` and so on will eventually accidentally clash once we flush out the actual Form components outside the conversations work. Thoughts?